packages/components-angular/projects/consumer-app/cypress/support/component-error-filter.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Sets up error monitoring for specific components.
+ * Captures console.error, uncaught errors, and unhandled rejections.
+ * @param componentNames - Array of component names to monitor
+ * @returns Object with errors array and setup function
+ */
+export function setupComponentErrorCapture(componentNames: string[]) {
+  const errors: string[] = [];
+  
+  const extractMessage = (arg: any): string => {
+    if (typeof arg === 'string') return arg;
+    if (arg?.message) return arg.message;
+    if (arg?.stack) return arg.stack;
+    try { return JSON.stringify(arg); } catch { return String(arg); }
+  };
+  
+  const isRelevant = (message: string): boolean => 
+    componentNames.some(name => message.toLowerCase().includes(name.toLowerCase()));
```

Comment:
Improve performance a little bit, by converting every string to lowercase only once.
Even if you in your implementation you already get lowercase (or kebab-case) componentNames...

```suggestion
  const errors: string[] = [];
  const lowerCaseComponentNames = componentNames.map(n => n.toLowerCase());
  
  const extractMessage = (arg: any): string => {
    if (typeof arg === 'string') return arg;
    if (arg?.message) return arg.message;
    if (arg?.stack) return arg.stack;
    try { return JSON.stringify(arg); } catch { return String(arg); }
  };
  
  const isRelevant = (message: string): boolean => {
    const lowerCaseMessage = message.toLowerCase();
    return lowerCaseComponentNames.some(n => lowerCaseMessage.includes(n));
  }
```